tests/unit/src/main/scala/tests/BaseAmmoniteSuite.scala

```diff
@@ -469,8 +470,21 @@ abstract class BaseAmmoniteSuite(scalaVersion: String)
         )
       )
       .asScala
-      .map { res =>
+      .flatMap { res =>
         val code = server.textContents(path)
-        TestHovers.renderAsString(code, Option(res), includeRange = true)
+        val result =
```

Comment:
I added a retry, because it can't figure out why is this happening. We've been having issues with hover anyway, so it can be just that, but more prominent in this situation. I also added additional logging that can help investigate.